wlauto/workloads/googlephotos/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -112,10 +105,9 @@ private void gesturesTest() throws Exception {
             int steps = pair.getValue().steps;
             int percent = pair.getValue().percent;
 
-            String runName = String.format(testTag + "_" + pair.getKey());
-            String gfxInfologName =  String.format(TAG + "_" + runName + "_gfxInfo.log");
+            String runName = String.format(TAG + "_" + testTag + "_" + pair.getKey());
+            String gfxInfologName =  String.format(runName + "_gfxInfo.log");
```

Comment:
Lose the TAG.   The results processing that happens next will parse based on testTag.  Having TAG will confuse it.